Lombiq.Hosting.Tenants.EnvironmentRobots/Extensions/IHostEnvironmentEnvironmentRobotsExtensions.cs

```diff
@@ -0,0 +1,12 @@
+using Lombiq.Hosting.Tenants.EnvironmentRobots.Models;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+
+namespace Lombiq.Hosting.Tenants.EnvironmentRobots.Extensions;
+public static class IHostEnvironmentEnvironmentRobotsExtensions
+{
+    public static bool IsProductionWithConfiguration(
+        this IHostEnvironment hostEnvironment,
+        IOptions<EnvironmentRobotsOptions> options) =>
+        options.Value.IsProduction ?? hostEnvironment.IsProduction();
```

Comment:
This line should be +1 indent compared to parameters. At least if I remember correctly.

Reply:
In this case I resolve this comment.
For me it was wierd to have a lambda method start on the same column as the parameters I saw indented versions of this in some project.